github_copilot_seats/triggers.py

```diff
@@ -0,0 +1,28 @@
+from autokitteh.slack import slack_client
+from seats import prune_idle_seats, find_idle_seats
+
+s = slack_client("slack_conn")
+
+
+def on_schedule() -> None:
+    print(prune_idle_seats())
+
+
+def on_slack_slash_command(event) -> None:
+    cmd = event.data.text
+    cid = event.data.channel_id
+
+    if cmd == "prune-idle-copilot-seats":
+        seats = prune_idle_seats()
+        msg = f"Engaged {len(seats)} new idle seats: {', '.join(get_logins(seats))}"
+        s.chat_postMessage(channel=cid, text=msg)
+    elif cmd == "find-idle-copilot-seats":
+        seats = find_idle_seats()
+        msg = f"Found {len(seats)} idle seats: {', '.join(get_logins(seats))}"
+        s.chat_postMessage(channel=cid, text=msg)
+    else:
+        s.chat_postMessage(channel=cid, text="Unrecognized command")
```

Comment:
`chat_postMessage` isn't necessarily bad, but note that slash commands are global (not limited to specific channels), and not visible to other users in that channel.

This means that there are 2 better options to respond to slash commands, choose whichever you prefer:

1. Use the user ID as the channel ID, so the app will send a (personal) DM to the user, regardless of the channel where they typed the command
2. Send an ephemeral message to the channel, instead of a regular message, so only the user sees the response

Reply:
Done